commit 7406b9d00324f352a00821db2d735cf09745dfc2 (refs/changes/27/127/1)
Author: Dave Pacheco <dap@joyent.com>
Date:   2016-07-21T15:45:25-07:00 (3 years, 3 months ago)
    
    MANTA-2951 update contribution guidelines to reflect use of cr.joyent.us

diff --git a/README.md b/README.md
index b8d2245..5948e18 100644
--- a/README.md
+++ b/README.md
@@ -5,7 +5,7 @@
 -->
 
 <!--
-    Copyright (c) 2015, Joyent, Inc.
+    Copyright (c) 2016, Joyent, Inc.
 -->
 
 # Manta: object storage with integrated compute
@@ -257,6 +257,19 @@ builds.
 
 ## Contributing to Manta
 
+To report bugs or request features, submit issues to the Manta project on
+Github.  If you're asking for help with Joyent's production Manta service,
+you should contact Joyent support instead.
+
+As of July, 2016, the Manta project is using Joyent's [code review
+system](https://cr.joyent.us) to manage all code changes.  Any registered GitHub
+user can submit changes through this system.  If you want to contribute a
+change, please see the [cr.joyent.us user
+guide](https://github.com/joyent/joyent-gerrit/blob/master/docs/user/README.md).
+If you're making a substantial change, you should contact developers on the
+mailing list or IRC first.  If you have any trouble with the contribution
+process, contact developers on the mailing list or IRC.
+
 Manta repositories use the same [Joyent Engineering
 Guidelines](https://github.com/joyent/eng/blob/master/docs/index.md) as
 the SDC project.  Notably:
@@ -279,12 +292,6 @@ areas, but generally speaking, style rules are arbitrary, while lint warnings
 identify potentially broken code.)  Repos sometimes have repo-specific lint
 rules, but this is less common.
 
-To report bugs or request features, submit issues to the Manta project on
-Github.  If you're asking for help with Joyent's production Manta service,
-you should contact Joyent support instead.  If you're contributing code, start
-with a pull request.  If you're contributing something substantial, you should
-contact developers on the mailing list or IRC first.
-
 
 ## Design principles
 
